vendor/github.com/google/uuid/CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ please explain why in the pull request description.
 ### Releasing
 
 Commits that would precipitate a SemVer change, as described in the Conventional
-Commits Specification, will trigger [`release-please`](https://github.com/google-github-actions/release-please-action)
+Commits Specification, will trigger [`release-please`](https://github.com/googleapis/release-please-action)
```

Comment:
Please remove this one replacement in the vendor directory as per vendor managed by tooling.